<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewClient</class>
 <widget class="QDialog" name="NewClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo cadastro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">NewClient {
background-image: url(:/images/background.jpeg);
}
</string>
  </property>
  <widget class="QPushButton" name="cadastrarButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelarButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="bankLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>-10</y>
     <width>431</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nomeLabel">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>150</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>false</kerning>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: white
</string>
   </property>
   <property name="text">
    <string>Nome completo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>262</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="enderecoLabel">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>180</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: white</string>
   </property>
   <property name="text">
    <string>Endereço</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineAddress">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>262</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineJob">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>262</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="profissaoLabel">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: white</string>
   </property>
   <property name="text">
    <string>Profissão</string>
   </property>
  </widget>
  <widget class="QLabel" name="salarioLabel">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>240</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: white</string>
   </property>
   <property name="text">
    <string>Salário</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineIncome">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>262</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
